spec/abci/abci++_app_requirements.md

```diff
@@ -615,6 +622,13 @@ If the Application sets value -1, consensus will:
 
 Must have `MaxBytes == -1` OR `0 < MaxBytes <= 100 MB`.
```

Comment:
If I remember correctly this -1 was introduced by @sergio-mena to allow for arbitrary block size or is there some other reason?  I know that in 0.34.x it is not allowed.

Reply:
Yes, this is probably only on `main`. 34 and 37 not for sure, v0.38 I'll have to double check when backporting.

Reply:
IIRC, this is on `main`, `v0.38.x`, `v0.37x` (as it tackles a corner case on `PrepareProposal`), but I might be mistaken

Reply:
In v0.37.x it is not implemented at consensus level, from this extract:

https://github.com/cometbft/cometbft/blob/v0.37.x/consensus/state.go#L1937

Reply:
In v0.38.x it is.

Reply:
So my memories weren't accurate